### DISCUSSION TITLE
Label positive TF regulator dotplot

### DISCUSSION THREAD

**Discussion by LacquerHed** [GreenleafLab/ArchR#1676] [Questions / Documentation]
I created the dotplot for the GeneIntegrationMatrix showing positive TF regulators which worked well, but I cant seem to figure out a way to label the red dots with the TF names? Thanks!

**Comment by rcorces** [comment]
You can manually add this to your ggplot object using `ggrepel`
https://cran.r-project.org/web/packages/ggrepel/vignettes/ggrepel.html

**Reply by LacquerHed** [reply]
Thanks for the tip! 

Im trying with the following however it seems to be labeling many points in the plots and not just the single red point in underlies. A little confused as to how to selectively label only the positive red dots. Thanks!

```
p <- ggplot(data.frame(corGIM_MM), aes(cor, maxDelta, color = TFRegulator)) +
  geom_point() + 
  theme_ArchR() +
  geom_vline(xintercept = 0, lty = "dashed") + 
  scale_color_manual(values = c("NO"="darkgrey", "YES"="firebrick3")) +
  xlab("Correlation To Gene Expression") +
  ylab("Max TF Motif Delta") +
  scale_y_continuous(
    expand = c(0,0), 
    limits = c(0, max(corGIM_MM$maxDelta)*1.05)
  )

p + geom_text_repel(data = data.frame(corGIM_MM), aes(), label = "Cd44") 
```

**Reply by rcorces** [reply]
https://ggrepel.slowkow.com/articles/examples.html#:~:text=Hide%20some%20of,non%2Dempty%20labels.
